java/overrides.js

```diff
@@ -81,4 +81,9 @@
 
     "should use regions padding": {skipEmit: true},
     "should use regions padding with vg": {skipEmit: true},
+
+    'regionId can be specified as part of checkSettings classic': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9690162)
```suggestion
    "regionId can be specified as part of checkSettings classic": {skipEmit: true},
```